File: Java/warmup2/FrontTimes.java
public String frontTimes(String str, int n) {
  String front = "";
  String result = "";
  
  if (str.length() < 3) {
    front += str;
  } else {
    front += (str.substring(0, 3));
  }
  
  for (int i = 0; i < n; i++) {
    result += front;
  }
  return result;
}

File: Python/Warmup_2/front_times.py
# Given a string and a non-negative int n, we'll say that the front of the
# string is the first 3 chars, or whatever is there if the string is less
# than length 3. Return n copies of the front; 

# front_times('Chocolate', 2) --> 'ChoCho'
# front_times('Chocolate', 3) --> 'ChoChoCho'
# front_times('Abc', 3) --> 'AbcAbcAbc'

def front_times(str, n):
  return (str[:3] * n)

print(front_times('Chocolate', 2))
print(front_times('Chocolate', 3))
print(front_times('Abc', 3))
